css/styles.css

```diff
@@ -1,137 +1,461 @@
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  justify-content: space-around;
+}
+
+#project1 {
+  order: 0;
+}
+
+#project2 {
+  order: 1;
+}
+
+#project3 {
+  order: 2;
+}
+
+#project4 {
+  order: 3;
+}
+
+#project1 .pimage {
+  margin-bottom: 5px;
+  width: 295px;
+  height: 220px;
+  background-image: url("../images/p1image.png");
+}
+
+#project2 .pimage {
+  margin-bottom: 5px;
+  width: 295px;
+  height: 220px;
+  background-image: url("../images/p2image.png");
+}
+
+#project3 .pimage {
+  width: 295px;
+  height: 220px;
+  background-image: url("../images/p3image.png");
+}
+
+#project4 .pimage {
+  width: 295px;
+  height: 220px;
+  background-image: url("../images/p4image.png");
+}
+
+.p-title {
+  width: 220px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  align-items: center;
+  text-align: left;
+  align-self: flex-start;
+}
+
+.p-title h2 {
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.details-list {
+  display: flex;
+  width: 295px;
+  height: 36px;
+  align-items: center;
+}
+
+.client {
+  width: 55px;
+  height: 16px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.spacedot {
+  width: 8px;
+  height: 8px;
+  background-image: url("../images/spacedot.png");
+}
+
+.role {
+  width: 90px;
+  height: 16px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.year {
+  width: 29px;
+  height: 16px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.p-description {
+  width: 295px;
+  height: 76px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.languages-list {
+  display: flex;
+  width: 295px;
+  height: 24px;
+  align-items: center;
+}
+
+.lang {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  width: auto;
+  height: 24px;
+  background: ##ebebff;
+  border-radius: 8px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.project-access {
+  box-sizing: border-box;
+  margin-top: 25px;
+  margin-bottom: 25px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  width: 132px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-family: Poppins, Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
```

Comment:
- [ ] According to the requirements, You should Implement the button interactions (enable, hover, etc.). Kindly, add the button interactions based on your Figma Design 👌.

![1](https://user-images.githubusercontent.com/45231276/176833382-1d1d826d-6c66-47ac-bec8-36427809918f.PNG)

![2](https://user-images.githubusercontent.com/45231276/176833730-b80a8ed8-05f1-4d93-b1d2-d78879b2be24.PNG)

Reply:
Thanks for the reminder! Implemented now.